### ISSUE TITLE
Chrome OS icon

### ISSUE THREAD

**Issue by lucasgabmoreno** [vinceliuice/grub2-themes#176]
Is there any posibbility to add Chrome OS icon?
Thanks!

**Comment by vinceliuice** [comment]
I don't the icon name should be

**Comment by lucasgabmoreno** [comment]
Hi @vinceliuice! 

There are two methods for installing Chrome OS in non Chromebook device: 
* Chrome OS Flex: https://chromeenterprise.google/intl/es-419/os/chromeosflex/ 
* Chrome OS Brunch: https://github.com/sebanc/brunch

Chrome OS Brunch allows to install into a partition for multi boot purpose (so it needs grub to load).
When installing it returns this grub entries:

```
menuentry "ChromeOS" --class "brunch" {
	rmmod tpm
	img_path=/chromeos.img
	img_uuid=88f52cc1-d9d3-4bce-a397-37e2af8a3315
	search --no-floppy --set=root --file $img_path
	loopback loop $img_path
	source (loop,12)/efi/boot/settings.cfg
	if [ -z $verbose ] -o [ $verbose -eq 0 ]; then
		linux (loop,7)$kernel boot=local noresume noswap loglevel=7 options=$options chromeos_bootsplash=$chromeos_bootsplash $cmdline_params \
			cros_secure cros_debug img_uuid=$img_uuid img_path=$img_path \
			console= vt.global_cursor_default=0 brunch_bootsplash=$brunch_bootsplash quiet
	else
		linux (loop,7)$kernel boot=local noresume noswap loglevel=7 options=$options chromeos_bootsplash=$chromeos_bootsplash $cmdline_params \
			cros_secure cros_debug img_uuid=$img_uuid img_path=$img_path
	fi
	initrd (loop,7)/lib/firmware/amd-ucode.img (loop,7)/lib/firmware/intel-ucode.img (loop,7)/initramfs.img
}

menuentry "ChromeOS (settings)" --class "brunch-settings" {
	rmmod tpm
	img_path=/chromeos.img
	img_uuid=88f52cc1-d9d3-4bce-a397-37e2af8a3315
	search --no-floppy --set=root --file $img_path
	loopback loop $img_path
	source (loop,12)/efi/boot/settings.cfg
	linux (loop,7)/kernel boot=local noresume noswap loglevel=7 options= chromeos_bootsplash= edit_brunch_config=1 \
		cros_secure cros_debug img_uuid=$img_uuid img_path=$img_path
	initrd (loop,7)/lib/firmware/amd-ucode.img (loop,7)/lib/firmware/intel-ucode.img (loop,7)/initramfs.img
}
```

So i guess --class brunch must be "brunch" icon name by now, maybe could add "chromeos" and 
"chromeos-flex" but it's not necessary.


I love your grub themes! 
Thanks!!!

**Comment by parker-norwood** [comment]
A chromeos icon would be awesome! I think a more obvious choice would be just "chromeos" for the file/class name, not "brunch".

**Comment by parker-norwood** [comment]
This was resolved in #191.
